packages/babel-compat-data/scripts/data/plugin-features.js

```diff
@@ -9,20 +9,6 @@ const es5 = {
   "transform-reserved-words": "Miscellaneous / Unreserved words",
 };
 
-// https://github.com/babel/babel/issues/11278
-// transform-parameters should run before object-rest-spread
```

Comment:
The relevant test (packages/babel-preset-env/test/fixtures/plugins-integration/issue-11278/input.mjs) still passes.